src/app/Console/Commands/Install.php

```diff
@@ -0,0 +1,88 @@
+<?php
+
+namespace Backpack\Base\app\Console\Commands;
+
+use Illuminate\Console\Command;
+use Symfony\Component\Process\Exception\ProcessFailedException;
+use Symfony\Component\Process\Process;
+
+class Install extends Command
+{
+    /**
+     * The name and signature of the console command.
+     *
+     * @var string
+     */
+    protected $signature = 'backpack:base:install';
+
+    /**
+     * The console command description.
+     *
+     * @var string
+     */
+    protected $description = 'Require dev packages and publish files for Backpack\Base to work';
```

Comment:
Don't think so. 
I don't think Laravel has options for translating command descriptions neither.

Reply:
Agreed. I don't think Laravel's command are translatable either. Since the only people who will see this are developers, English is a must. We don't even have docs in different languages, so I say English should be the only supported language for commands.